content/blog/2021-09-21-collaborative-experiments.md

```diff
@@ -0,0 +1,144 @@
+[the Getting Started doc](https://dvc.org/doc/start).
+
+## Setting up your DVC remotes
+
+When you want to share the progress you've made with training your model, that
+usually means you need to find a way to bundle the code, data, and
+hyperparameters. This could be a complicated process if you're working with GBs
+worth of data or you have a large number of hyperparameters.
+
+That's one of the uses for DVC and why we'll be working with remotes. To start
+with, make sure your GitHub remote is configured correctly. It should use the
+SSH version of the URL. This is so DVC can authenticate the pushes and pulls
+from GitHub it needs as part of experiment sharing.
+
+The way DVC works is by storing custom Git refs in your repo with metadata that
+defines the experiment. You can learn more about how DVC uses custom Git refs in
+[this post](https://dvc.org/blog/experiment-refs).
+
+Next, you'll need to set up a remote to your data location. This could be an AWS
+S3 bucket, a Google Drive, or Blob Storage in Azure. For this example, we'll be
+using a Google Drive folder as the remote to handle data storage.
+
+Now that you know what we're doing, let's run the command to set up our DVC
+remote.
+
+```dvc
+dvc remote add -d cloud_remote gdrive://1k6aUYWphOulJlXgq4XbfKExWGyymTpEl
+```
+
+This adds the remote storage named `cloud_remote` for DVC to track and we'll be
+able to push and pull the exact code and data to reproduce any experiment from
+end to end. With your Git remote and DVC remote in place, you can start pulling
+data and experiments from the cloud to your local machine.
+
+You can learn more about the Google Drive setup in
+[the docs](https://dvc.org/doc/command-reference/remote/add). The main thing you
+need to know is that you need the folder id for the remote location you want to
+set up. You can find that id in the URL for that folder. It'll look similar to
+this.
+
+```dvc
+https://drive.google.com/drive/folders/1k6aUYWphOulJlXgq4XbfKExWGyymTpEl?usp=sharing
+```
+
+## Listing your remote experiments
+
+When you're working with a team on an existing project, you might want to see
+the experiments already in the remotes so you know what's available. You can
+take a look at all of the experiments with the following command.
+
+```dvc
+dvc exp list origin --all
+```
+
+You'll get a list of all of the experiments across different Git branches that
+have been pushed with DVC. The output will look similar to this.
+
+```dvc
+21784fa:
+        exp-c8dcf
+main:
+        exp-b3667
+        exp-d382a
+```
+
+Now you'll be able to pick which experiment you want to reproduce and start
+testing with.
+
+## Pulling experiments
+
+If you're picking up an existing project, there will likely be a specific
+experiment you'll get started with. To pull an experiment to your local machine,
+you'll need an experiment id for the following command.
+
+```dvc
+dvc exp pull origin exp-b3667
```

Comment:
Is there a read-only remote that can be configured on this repo for users to pull the data associated with this experiment? This also might complicate things a little since users might not be able to use a single default remote, but instead may need to pull from one remote and push to another.

Reply:
🤔 That's a good point. Is there a downside of using a Google Drive folder for this? That's what I was planning on doing, but there weren't any plans to maintain it or than clearing it out every now and then.

Reply:
Yeah, I guess the downsides would be managing the size, plus the lack of control over what users push. What do others think?

Reply:
I remember a while ago there was a mention of a `s3://dvc-public` remote. I'm checking in the infrastructure channel to see if that's an option.